src/coreclr/jit/llvmlower.cpp

```diff
@@ -998,6 +1003,11 @@ void Llvm::lowerCall(GenTreeCall* callNode)
         {
             CurrentRange().Remove(CurrentRange().LastNode(), /* markOperandsUnused */ true);
         }
+
+        if (!CurrentBlock()->KindIs(BBJ_THROW))
+        {
+            _compiler->fgConvertBBToThrowBB(CurrentBlock());
+        }
```

Comment:
Some blocks weren't marked `BBJ_THROW` even as they has always-throwing calls in them.